src/change_stream.ts

```diff
@@ -397,6 +381,7 @@ export class ChangeStreamCursor extends Cursor<AggregateOperation, ChangeStreamC
   hasReceived?: boolean;
   resumeAfter: ResumeToken;
   startAfter: ResumeToken;
+  [kCursorStream]?: CursorStream;
```

Comment:
I think this belongs on the `ChangeStream` type. This type is a specialization on `Cursor` which is solely focused on isolating initialization and property tracking for the cursor, I don't think we should be using this type to store private members of the parent `ChangeStream`. 

When a user uses the `ChangeStream` as an event emitter (listening to the `change` event), then we want to create a singular internal cursor stream to listen to - this is differentiated from calling `stream()` which can happen many times and create many streams for a user to consume, backed by the same cursor.